src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -1436,12 +1473,30 @@ public static void Remove_Invalid()
             var builder = new StringBuilder("Hello");
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Remove(-1, 0)); // Start index < 0
             Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(0, -1)); // Length < 0
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Parameter name changes in error messages are breaking, not currently backported.")]
+        public static void Remove_Index_Invalid()
+        {
+            var builder = new StringBuilder("Hello");
 
             Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(6, 0)); // Start index + length > 0
```

Comment:
Hmm, looks like this but it's commented out:
https://github.com/dotnet/corefx/blob/bffef76f6af208e2042a2f27bc081ee908bb390b/src/Common/tests/System/RuntimeDetection.cs#L15